plugins/catalog-backend/src/catalog/types.ts

```diff
@@ -41,12 +42,23 @@ export type EntitiesCatalog = {
 // Locations
 //
 
+export type status = {
```

Comment:
Gotta be uppercase first char. And "status" is probably too generic.

Reply:
- [x] fix type name